stories/GOTerm.stories.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import GoTerms from '../src/components/GoTerms';
+
+import Provider from './Provider';
+import configureStore from './configuedStore.js';
+
+const store = configureStore();
+
+const withProvider = (story) => <Provider store={store}>{story()}</Provider>;
+
+export default {
+  title: 'InterPro UI/GOTerms',
+  decorators: [withProvider],
+};
+
+const data = [
+  {
+    category: {
+      code: 'P',
+      name: 'biological_process',
+    },
+    identifier: 'GO:0019058',
+    name: 'viral life cycle',
+  },
+  {
+    category: {
+      code: 'C',
+      name: 'cellular_component',
+    },
+    identifier: 'GO:0042025',
+    name: 'host cell nucleus',
+  },
+];
+export const Basic = () => (
+  <GoTerms terms={data} type={'entry'} db={'interpro'} />
```

Comment:
maybe display this with different values for type and db... or maybe use knobs?

Reply:
I have been looking for examples for this, any suggestion?

Reply:
Mhh... you got me here... it seems that it looks the same with different values of `type`:
* https://www.ebi.ac.uk/interpro/protein/UniProt/A0A000/
* https://www.ebi.ac.uk/interpro/entry/InterPro/IPR004839/
Where the only difference is the Tooltip.

And I check in the database and the only db with go terms is `interpro`, so not sure how useful is the `db` attribute.

Reply:
yes, I did check for all the other DBs. If it is going to be only interpro [this check ](https://github.com/ProteinsWebTeam/interpro7-client/blob/master/src/components/GoTerms/index.js#L65) is not necessary right

Reply:
Not at the moment. But some member DB have their own GO annotations, is only that we aren't importing them now. 
So I would leave that code there, just in case later we decide to include their go annotations.